src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/IOpenIdConnectEvents.cs

```diff
@@ -36,6 +36,11 @@ public interface IOpenIdConnectEvents : IRemoteAuthenticationEvents
         Task RedirectToIdentityProviderForSignOut(RedirectContext context);
 
         /// <summary>
+        /// Invoked before redirecting to the post logout redirect uri.
+        /// </summary>
+        Task RedirectToPostLogoutRedirectUri(RedirectContext context);
```

Comment:
If the bug we file is to change the interface, please mark it as `breaking-change`.